homeassistant/components/climate/isy994.py

```diff
@@ -0,0 +1,321 @@
+
+    @property
+    def target_temperature_step(self):
+        """Return the supported step of target temperature."""
+        return 1
+
+    @property
+    def target_temperature_high(self):
+        """Return the highbound target temperature we try to reach."""
+        return self._target_temp_high
+
+    @property
+    def target_temperature_low(self):
+        """Return the lowbound target temperature we try to reach."""
+        return self._target_temp_low
+
+    @property
+    def is_on(self):
+        """Return if the unit is on or off based on operation status."""
+        if self._current_status not in [STATE_OFF, STATE_IDLE, STATE_UNKNOWN]:
+            return True
+        return False
+
+    @property
+    def fan_list(self):
+        """Return the list of available fan modes."""
+        return [STATE_AUTO, STATE_ON]
+
+    @property
+    def current_fan_mode(self) -> str:
+        """Return the current fan mode ie. auto, on."""
+        return self._fan_mode
+
+    @property
+    def supported_features(self):
+        """Return the list of supported features."""
+        return (SUPPORT_OPERATION_MODE |
+                SUPPORT_FAN_MODE |
+                SUPPORT_TARGET_TEMPERATURE_HIGH |
+                SUPPORT_TARGET_TEMPERATURE_LOW)
+
+    @property
+    def min_temp(self):
+        """Return the minimum temperature."""
+        return convert_temperature(DEFAULT_MIN_TEMP, TEMP_CELSIUS,
+                                   self.temperature_unit)
+
+    @property
+    def max_temp(self):
+        """Return the maximum temperature."""
+        return convert_temperature(DEFAULT_MAX_TEMP, TEMP_CELSIUS,
+                                   self.temperature_unit)
+
+    @property
+    def device_state_attributes(self):
+        """Return the device specific state attributes."""
+        data = {
+            ATTR_CURRENT_TEMPERATURE: self.current_temperature,
+            ATTR_CURRENT_HUMIDITY: self.current_humidity,
+            ATTR_FAN_LIST: self.fan_list,
+            ATTR_FAN_MODE: self.current_fan_mode,
+            ATTR_OPERATION_LIST: self.operation_list,
+            ATTR_OPERATION_MODE: self.current_operation,
+            ATTR_TARGET_TEMP_HIGH: self.target_temperature_high,
+            ATTR_TARGET_TEMP_LOW: self.target_temperature_low
+        }
+        return data
+
+    def set_temperature(self, **kwargs):
+        """Set new target temperature."""
+        if 'target_temp_high' in kwargs and kwargs['target_temp_high'] != \
+                self._target_temp_high:
+            self._node.climate_setpoint_cool(int(kwargs['target_temp_high']))
+            self._target_temp_high = kwargs['target_temp_high']
+        if 'target_temp_low' in kwargs and kwargs['target_temp_low'] != \
+                self._target_temp_low:
+            self._node.climate_setpoint_heat(int(kwargs['target_temp_low']))
+            self._target_temp_low = kwargs['target_temp_low']
+        self.schedule_update_ha_state()
```

Comment:
These are not needed, this is a polling device so HA will already do this after calling this method.

Reply:
ISY is a Local Push and the states on the front end did not update correctly when I removed these lines.